### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Rustamxon7**
### Math Magicians: tests

All requirements are complete: 

- Unite tests written for `operate.js` and `calculate.js`. ✅ 
- All components are tested and created a snapshot for each of them. ✅ 

Feel free to review -🧑🏻‍💻 - our unite testing and let us know if there are any changes to do. 🤓 -😉

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import React from 'react';
-import Calculator from './components/Calculator';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+// import Calculator from './components/Calculator';
```
Comment (wandji20) [inline src/App.js:3]: - [ ] It will be nice to remove all commented code from your code-base.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @Rustamxon7  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Very descriptive commit messages and pull request summary
:heavy_check_mark: Excellently crafted README 
:heavy_check_mark: Snapshots test suites are passing :100: 
:heavy_check_mark: Unit tests for calculator.js and operate.js :100: 


## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved

- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Rustamxon7) [reply src/App.js]: Thank you for reminding me about it.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Rustamxon7, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**